routers/web/repo/webhook.go

```diff
@@ -445,7 +456,6 @@ func matrixHookParams(ctx *context.Context) webhookParams {
 		Meta: &webhook_service.MatrixMeta{
 			HomeserverURL: form.HomeserverURL,
 			Room:          form.RoomID,
-			AccessToken:   form.AccessToken,
```

Comment:
Is this a break change?

Reply:
It might sound stupid, but I could not find documentation of gitea compatibility, hence I don't know what is defined as a "breaking change".

- If gitea was meant to be used as a go package, yes, any deletion of an exported field would be a breaking change (so #20729 was a breaking change, since `utils.IsValidSlackChannel` was removed?).

- If you care about API compatibility, this change is not breaking, since it only adds an optional `authorization_header` field (`templates/swagger/v1_json.tmpl`).

- Regarding the web frontend, this change is breaking, but so are most changes which touches forms, I guess?

Reply:
Yes, I mean before there is an input in the form and then it disappeared.

Reply:
Ok, in this case yes, the following scenario would be broken:
- someone is adding (or editing) a **matrix webhook**, while the gitea upgrade happens (form loaded before the upgrade, submitted after the upgrade)

In this case the given `Access token` will be ignored and the `Authorization` header will be empty.

I guess the user will see it quite fast, since no webhook task will work.

Reply:
@lunny should I do something about it? (mention it somewhere?)

Reply:
Maybe left it as before if it will not affect this PR.